src/Goerli-DssSpell.t.sol

```diff
@@ -310,15 +342,17 @@ contract DssSpellTest is GoerliDssSpellTestBase {
         uint256 castTime = block.timestamp + pause.delay();
         hevm.warp(castTime);
         cureSpell.cast();
+        (, , , uint48 toc) = oracle.ilks(ilk);
+        assertEq(uint256(toc), 0);
```

Comment:
```suggestion
```

Those assertions will never be called because `cureSpell.cast()` will always fail.